### ISSUE TITLE
zenodo repo snapshot

### ISSUE THREAD

**Issue by AngryMaciek** [zavolanlab/SCUREL#2]
@dominikburri : consider uploading a permanent snapshot of the repository to [zenodo](https://zenodo.org/)?
That's ok even now; you can later put out a next version (release), if necessary.

**Comment by dominikburri** [comment]
Yes I think that could be a good idea. Will follow up on this :)

**Comment by AngryMaciek** [comment]
If you do remember to add a fancy shield to the README, like ex. [here](https://github.com/zavolanlab/bindz/blob/dev/README.md) :)
